### PR TITLE
Merge cron's separate debug property with SP_DEBUG

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

**Comment by sendtogeo** [comment]
Thanks for the update. I will add it soon this week. Needs to copy my
changes to git first.



Regards,
Geo Varghese
www.seofreetools.net

File: controllers/cron.ctrl.php [hunk 1]
```diff
@@ -25,11 +25,16 @@ class CronController extends Controller {
     
 	var $cronList;			    // the array includes all tools avialable for cron
 	var $repTools;			    // the array includes all tools avialable for report generation
-	var $debug = true;		    // to show debug message or not
+	var $debug = false;		    // to show debug message or not
 	var $layout = 'ajax';       // ajax layout or not
 	var $timeStamp;             // timestamp for storing reports
 	var $checkedKeywords = 0;   // the number of keywords checked in cron, this is used for split cron execution feature       	
 	
+	# initialize preperties
```
Comment (szepeviktor) [inline controllers/cron.ctrl.php]: A typo here!

Comment (sendtogeo) [inline controllers/cron.ctrl.php]: Done. Thanks for the notification.



Regards,
Geo Varghese
www.seofreetools.net

Comment (szepeviktor) [inline controllers/cron.ctrl.php]: Thanks.

**Pull request merged**